text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,376 @@ nit-picking the "gotchas" in code examples. :heart:
 
 # Summary
 
-Brief explanation of the feature.
+Migrate Headless-styles to use [Panda CSS](https://panda-css.com/) under the hood via
+a new public Pando preset configuration option called `@pluralsight/panda-preset`.
+
+This will allow both Headless-styles and the React lib to be more flexible and
+configurable for future use-cases by leveraging the power of Panda CSS. Additionally,
+it will increase the overall performance of the React lib by reducing the amount of
+CSS that is shipped to the client (via Atomic-CSS).
```

Comment:
Just want to clarify in relation to [this discussion](https://github.com/pluralsight/pando/discussions/1658) that Panda removes the use of / need for global CSS from headless styles?

Reply:
@timkinnane correct, that will now be built into what will be the `@pluralsight/panda-preset` so no need to load normalize or depend on custom CSS loaders for Pando.